db/migrate/20150521121315_rename_config_template_to_provisioning_template.rb

```diff
@@ -5,15 +5,15 @@ def up
     old_name = 'ConfigTemplate'
     new_name = 'ProvisioningTemplate'
 
-    Template.update_all "type = '#{new_name}'", "type = '#{old_name}'"
-    Audit.update_all "auditable_type = '#{new_name}'", "auditable_type = '#{old_name}'"
-    TaxableTaxonomy.update_all "taxable_type = '#{new_name}'", "taxable_type = '#{old_name}'"
-    Permission.update_all "resource_type = '#{new_name}'", "resource_type = '#{old_name}'"
+    Template.where("type = '#{old_name}'").update_all("type = '#{new_name}'")
```

Comment:
Nitpick, can we use hashes in this file now rather than string interpolation?  It's easier to read and less error-prone IMHO.

e.g. `Template.where(:type => old_name).update_all(:type => new_name)`